etc/dockerfiles/stack-build/lts-8.0/Dockerfile

```diff
@@ -43,6 +43,7 @@ RUN stack --system-ghc --resolver=$LTS_SLUG --local-bin-path=/usr/local/bin inst
 #
 
 RUN wget -O- "https://github.com/fpco/pid1/releases/download/pid1%2F$PID1_VERSION/pid1-$PID1_VERSION-linux-x86_64.tar.gz" |tar xzf - -C /usr/local
+RUN chown -R root:root /usr/local/sbin
```

Comment:
@borsboom, yeah, adding `chown` to the same layer makes total sense. Changing the ownership of `pid1` only, wouldn't be sufficient though, since `/usr/local/sbin` ownership is also set to `501:staff` (it seems to be created during the untar process). Considering that `sbin` contents is usually owned by root and especially since the folder is initially empty or nonexistent in that image, doing `chown -R` seems to be reasonable, but I can see how it can have an undesired effect in case of future customizations. In that case I can change it to:

```
RUN wget -O- "https://github.com/fpco/pid1/releases/download/pid1%2F$PID1_VERSION/pid1-$PID1_VERSION-linux-x86_64.tar.gz" | tar xzf - -C /usr/local && \
     chown root:root /usr/local/sbin && \
     chown root:root /usr/local/sbin/pid1
```
Would that be better?